src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -331,6 +332,42 @@ public function move($path, $target)
         return rename($path, $target);
     }
 
+    /**
+     * Create a file from a base64 image
+     *
+     * @param  string  $base64File
+     * @return string
+     */
+    public function fromBase64(string $base64File)
+    {
+        if (!$this->isBase64File($base64File)) {
+            throw new InvalidBase64FileException();
+        }
+
+        if (strpos($base64File, 'data:image') !== false) {
```

Comment:
This might be simplified to:
```suggestion
        if (str_contains($base64File, 'data:image')) {
```